src/compliment/sources/class_members.clj

```diff
@@ -114,19 +123,25 @@
 (defn members-candidates
   "Returns a list of Java non-static fields and methods candidates."
   [prefix ns context]
-  (when (class-member-symbol? prefix)
-    (let [prefix (subs prefix 1)
-          inparts? (re-find #"[A-Z]" prefix)
-          klass ^{:lite nil} (try-get-object-class ns context)]
-      (for [[member-name members] (get-all-members ns klass)
+  (when-let [[klass-name prefix] (class-member-symbol? prefix)]
+    (let [prefix      (or prefix "")
+          qualified?  (some? klass-name)
+          inparts?    (re-find #"[A-Z]" prefix)
+          klass       ^{:lite nil} (if qualified?
```

Comment:
`^{:lite nil}` should be for `try-get-object-class`. This way, compliment-lite will still suggest `Class/.member ` candidates.